Gopkg.lock

```diff
@@ -25,21 +31,51 @@
   packages = ["blockchain","btcec","chaincfg","chaincfg/chainhash","database","txscript","wire"]
   revision = "8e7c0427fee5d4778c5d4eb987150369e3ca1d0e"
 
+[[projects]]
+  branch = "master"
+  name = "github.com/btcsuite/btcd"
+  packages = ["btcec"]
+  revision = "962a206e94e9151fe41bbd6d6464af4ba7168f50"
```

Comment:
Please remove `github.com/btcsuite/btcd`, we are using a forked version of it (which is also included in the `Gopkg.lock`). I tried to build it without `github.com/btcsuite/btcd` and it worked fine.